pkg/extensions/search/cve/cve.go

```diff
@@ -19,7 +20,7 @@ import (
 type CveInfo interface {
 	GetImageListForCVE(repo, cveID string) ([]common.TagInfo, error)
 	GetImageListWithCVEFixed(repo, cveID string) ([]common.TagInfo, error)
-	GetCVEListForImage(repo, tag string, pageinput PageInput) ([]cvemodel.CVE, PageInfo, error)
+	GetCVEListForImage(repo, tag string, pageinput PageInput, searchedCVE *string) ([]cvemodel.CVE, PageInfo, error)
```

Comment:
I think we should convert it from pointer to string before calling this function. For consistency with the parameters `repo` and `tag`. Also reorder the parameters for `pageinput` to be last.